packages/core/test/moduleuserapp/export-func.js

```diff
@@ -1,3 +1,3 @@
 module.exports = (z, bundle) => {
-  return [{ id: 1234 }] || bundle;
```

Comment:
Calling out this change I did to fix `no-constant-binary-expression` eslint error